docs/docs/forms.mdx

```diff
@@ -142,41 +140,12 @@ write stories or rules for this case.
 
 ### Slot Mappings
 
-Rasa Open Source comes with four predefined mappings to fill the slots of a form
-based on the latest user message. Please see
-[Custom Slot Mappings](./forms.mdx#custom-slot-mappings) if you need a custom function
-to extract the required information.
-
-#### from_entity
-
-The `from_entity` mapping fills slots based on extracted entities.
-It will look for an entity called `entity_name` to fill a slot `slot_name`.
-If `intent_name` is `None`, the slot will be filled regardless of intent name.
-Otherwise, the slot will only be filled if the user's intent is `intent_name`.
-
-If `role_name` and/or `group_name` are provided, the role/group
-label of the entity also needs to match the given values. The slot mapping will not
-apply if the intent of the message is `excluded_intent`. Note that you can
-also define lists of intents for the parameters `intent` and `not_intent`.
+:::caution Changed in 3.0
+As of 3.0, [slot mappings](./domain.mdx#slot-mappings) can only be defined in the slots section of the domain.
+Please follow the migration guide to update your assistant.
+:::
 
-```yaml-rasa
-entities:
-- entity_name
-slots:
-  slot_name:
-    type: any
-    mappings:
-    - type: from_entity
-      entity: entity_name
-      role: role_name
-      group: group name
-      intent: intent_name
-      not_intent: excluded_intent
-forms:
-  your_form:
-    required_slots:
-        - slot_name
-```
+#### Unique entity mapping matching
 
 In `from_entity` mapping, when an extracted entity uniquely maps onto a slot,
 the slot will be filled even if this slot wasn't requested by the form.
```

Comment:
I know you didn't write this, but could you add a short explanation/example of what "uniquely maps" means here?